[2024-05-02T09:16:45.975-0500] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-02T09:16:45.983-0500] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scenario_two_v3.second_task manual__2024-05-02T14:14:41.582691+00:00 [queued]>
[2024-05-02T09:16:45.986-0500] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scenario_two_v3.second_task manual__2024-05-02T14:14:41.582691+00:00 [queued]>
[2024-05-02T09:16:45.986-0500] {taskinstance.py:2303} INFO - Starting attempt 2 of 6
[2024-05-02T09:16:45.991-0500] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): second_task> on 2024-05-02 14:14:41.582691+00:00
[2024-05-02T09:16:45.994-0500] {standard_task_runner.py:63} INFO - Started process 39459 to run task
[2024-05-02T09:16:45.998-0500] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'scenario_two_v3', 'second_task', 'manual__2024-05-02T14:14:41.582691+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dotnet_dag.py', '--cfg-path', '/var/folders/14/31_1v10s5nvcqyswsggcn5vh0000gn/T/tmpiys334vn']
[2024-05-02T09:16:45.999-0500] {standard_task_runner.py:91} INFO - Job 31: Subtask second_task
[2024-05-02T09:16:46.015-0500] {task_command.py:426} INFO - Running <TaskInstance: scenario_two_v3.second_task manual__2024-05-02T14:14:41.582691+00:00 [running]> on host nestors-macbook.attlocal.net
[2024-05-02T09:16:46.048-0500] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nestor' AIRFLOW_CTX_DAG_ID='scenario_two_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T14:14:41.582691+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-02T14:14:41.582691+00:00'
[2024-05-02T09:16:46.048-0500] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-02T09:16:46.058-0500] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/14/31_1v10s5nvcqyswsggcn5vh0000gn/T
[2024-05-02T09:16:46.058-0500] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dotnet run --project /Users/nestoralfaro/Desktop/pbi-scenario-two/dotnetapp/dotnetapp.csproj']
[2024-05-02T09:16:46.062-0500] {subprocess.py:86} INFO - Output:
[2024-05-02T09:16:46.872-0500] {subprocess.py:93} INFO - Hello, World!
[2024-05-02T09:16:46.882-0500] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-02T09:16:46.883-0500] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-02T09:16:46.893-0500] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=scenario_two_v3, task_id=second_task, execution_date=20240502T141441, start_date=20240502T141645, end_date=20240502T141646
[2024-05-02T09:16:46.924-0500] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-02T09:16:46.930-0500] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-02T09:16:46.931-0500] {local_task_job_runner.py:222} INFO - ::endgroup::
